content/en/blog/call-for-blogs.md

```diff
@@ -0,0 +1,42 @@
+# Call for blog contributions
+
+This is a constantly updated list of the blogs that the sig-docs is interested in.
+
+This will list the blogs along with abstract that can be furthre used for gathering information.
+
+---
```

Comment:
When someone looks at the list below, what do we want them to understand?
From the submitted issue and the explanation provided on Slack, I assume we want the users to know about the different types of blogs we publish or that they can submit content for. If that's the case, the one-line description provided against each listing is inadequate.

Reply:
Here's what I concluded from the issue:

As a [reference link](https://github.com/keras-team/keras-io/blob/master/call_for_contributions.md) provided on the issue, the blogs were selected with an intent to provide a little description of the _important blogs_.  Hence I listed some which include v1.26 features, etc. Do you think there's a need to change the number of blogs mentioned?

Reply:
So, let me rephrase this in a way you'd understand better. Release blogs, feature blogs, etc., are written by folks within the community who are working on the releases and features. They are submitted as a part of the release, and we DO NOT invite contributions externally for them, i.e., they are being worked on by active contributors within the community. 
I am unsure how the Keras project functions, but to write a call-out, you need to tailor the call-out specific to our community to make it relevant. Which is why, it is okay to include one/two release blog examples in the call-out but not have an entire listing about release-related blogs since there are other types of blogs we invite contributions for.

Reply:
To understand the other types, you will need to go beyond the surface level of the first page for blogs and do a bit of research.